src/evaluation/liveRec/buildIdentificationBBS.m

```diff
@@ -0,0 +1,14 @@
+function bbs = buildIdentificationBBS(sim,idModels,labels,onOffsets)
+
+bbs = BlackboardSystem(1);
+bbs.setRobotConnect(sim);
+bbs.setDataConnect('AuditoryFrontEndKS');
+for ii = 1 : numel( idModels )
+    idKss{ii} = bbs.createKS('IdentityKS', {idModels(ii).name, idModels(ii).dir});
+    idKss{ii}.setInvocationFrequency(50);

```

Comment:
It mainly depends on the blocksize of the robotConnect, i.e. in case of the jido interface, it seems to be written somewhere in the bass object. The inverse of this blocksize is the max functional invocation frequency. 

However, I guess something like 4Hz should be enough -- this means you are effectively operating with a 0.25s shift of the blocks. (50Hz would be a 20ms shift...)